models/feedItem.js

```diff
@@ -0,0 +1,18 @@
+var mongoose = require( 'mongoose' );
+
+// create feedItem schema
+var options = { discriminatorKey: 'type' };
+
+var feedItemSchema = new mongoose.Schema({
+    _id: {
+        $oid: String
+    },
+    title: String,
+    description: String,
+    index: Number
+}, options);
+
+// create model
+var FeedItem = mongoose.model('feeditems', feedItemSchema);
```

Comment:
FeedItem is the model object. Standard is to capitalize that as opposed to the scheme which is camel cased. 

'feeditems' is the name of the actual collection in our DB. Standard is to do all lower case because it's case sensitive and sending capital letters through the node mongoDB driver and across http can get screwy sometimes.